pydm/data_plugins/local_plugin.py

```diff
@@ -25,10 +25,6 @@ def __init__(self, channel, address, protocol=None, parent=None):
         self.connected = False
         self._configure_local_plugin(address)
 
-    def add_listener(self, channel):
```

Comment:
I removed the `add_listener` that we added last time, because I had it previously defined in here. Also, I did not add the `self._configure_local_plugin(channel.address)` back because it was giving me problems, and I think it should get called anytime we make a connection anyway correct? Please let me know if I should add it back....